frontend/src/components/CreateSeason/index.tsx

```diff
@@ -168,11 +201,15 @@ class _CreateSeason extends Component<Props, IState> {
 }
 
 const mapStateToProps = (state: IStore) => ({
+  currentSeason: state.applicationSeason.currentSeason,
+  id: state.applicationSeason.id,
   submitted: state.applicationSeason.submitted,
```

Comment:
The fields above could be combined into applicationSeason and sent as one in props, but that might not be necessary

Reply:
Agreed, but I preferred splitting them, but can combine them if others find it better as well.

Reply:
I think it's fine, I'll approve